CONTRIBUTING.md

```diff
@@ -43,15 +43,26 @@ To create a release for the latest version:
 ```
 git checkout develop
 git pull
+# check with git log that the branch is in the state which you want to release
 git checkout master
 git pull
-git merge develop --ff-only
```

Comment:
I am not sure myself if we really want to force it. Leaving it out creates a fast-forward merge anyways if we worked cleanly. Adding this option makes the merge fail if there is no fast-forward possible. If we want FFs, I'd prefer leaving it in the contributing file.

Reply:
My understanding was `--ff-only` is no longer possible in our scenario?

Reply:
Why would that be? We merge everything into master, which would be only FFs.